spec/services/recurring_contribution/transactions/find_or_create_spec.rb

```diff
@@ -21,6 +21,20 @@
         allow(Pagarme::API).to receive(:find_transaction).and_return(transaction)
       end
 
+      context "and the transaction's subscription has a current transaction" do
+        let!(:latest_current_transaction) { create(:transaction, current: true, subscription: subscription) }
+
+        before { subject }
+
+        it "sets the current attribute value of that transaction to false" do
+          expect(latest_current_transaction.reload.current).to eq false
+        end
+
+        it "creates the new transaction with true as the current attribute value" do
+          expect(created_transaction.current).to eq true
```

Comment:
Não da pra fazer?

```Ruby
it "..." do
  subject

  expect(created_transaction).to be_current
end
```

Usando o subject dentro do `it`, vc não precisa mais do `before`.